java/src/main/resources/arguments.yaml

```diff
@@ -43,6 +43,8 @@ NUMBER_OF_EVENTS_IN_FETCHREQUEST: null
 REPLAY_PRESET: null
 # Replay ID in ByteString
 REPLAY_ID: null
+# Flag to enable/disable bitmap field processing in Subscribe and ManagedSubscribe examples for CDC events (default: false)
```

Comment:
Edit:
`# Flag to enable/disable bitmap ChangeEventHeader field processing in Subscribe and ManagedSubscribe examples for change data capture events (default: false)`